src/bootstrap/lib.rs

```diff
@@ -754,6 +754,9 @@ impl Build {
             LlvmLibunwind::System => features.push_str(" system-llvm-libunwind"),
             LlvmLibunwind::No => {}
         }
+        if target.contains("xous") {
+            features.push_str(" compiler-builtins-mem");
```

Comment:
You could enable this directly in compiler-builtin's build script. That way `cargo build -Zbuild-std` will also get this enabled automatically.

Reply:
That's a really good suggestion. I've created https://github.com/rust-lang/compiler-builtins/pull/548 that does this, and I've tested that it works locally.

Reply:
With 0.1.101 released, I've removed the patch and updated `Cargo.lock` to pull in the latest version.